core_impl/src/main/java/io/opencensus/implcore/stats/StatsComponentImplBase.java

```diff
@@ -59,6 +59,7 @@ public StatsCollectionState getState() {
   }
 
   @Override
+  @Deprecated
```

Comment:
Don't need to deprecated the internal function.

Reply:
I think this method still needs to be marked as Deprecated, since it's a public method (though it's in impl) and users can call it if they have compile-time dependency on opencensus-impl.

Reply:
Yes, but we do not promise/guarantee any backwards compatibility in this package. So we can remove this at any moment.

Reply:
Make sense to me. Removed.